Sources/Plasma/PubUtilLib/plAvatar/plArmatureMod.h

```diff
@@ -233,7 +233,9 @@ class plArmatureMod : public plArmatureModBase
     virtual void PanicLink(bool playLinkOutAnim = true);
     virtual void PersonalLink();
 
-    virtual bool ToggleDontPanicLinkFlag() { fDontPanicLink = fDontPanicLink ? false : true; return fDontPanicLink; }
+    bool ToggleDontPanicLinkFlag() { fDontPanicLink = fDontPanicLink ? false : true; return fDontPanicLink; }
```

Comment:
The `virtual` part of the definition was removed, is that intentional? If so, why?

Reply:
No one is overriding this function in a subclass, so it doesn't need to be `virtual` (improves performance).